packages/style-loader/index.js

```diff
@@ -40,19 +31,22 @@ module.exports = (neutrino, opts = {}) => {
   const rules = [options];
 
   if (options.modules) {
-    rules.push(merge(options, {
-      test: options.modulesTest,
-      ruleId: `${options.ruleId}${options.modulesSuffix}`,
-      styleUseId: `${options.styleUseId}${options.modulesSuffix}`,
-      cssUseId: `${options.cssUseId}${options.modulesSuffix}`,
-      hotUseId: `${options.hotUseId}${options.modulesSuffix}`,
-      extractId: `${options.extractId}${options.modulesSuffix}`,
-      css: {
-        modules: options.modules,
-        importLoaders: 1
-      }
-    }));
-  }
+    rules.push(
+      merge(options, {
+        test: options.modulesTest,
+        exclude: options.modulesExclude,
+        ruleId: `${options.ruleId}${options.modulesSuffix}`,
+        styleUseId: `${options.styleUseId}${options.modulesSuffix}`,
+        cssUseId: `${options.cssUseId}${options.modulesSuffix}`,
+        hotUseId: `${options.hotUseId}${options.modulesSuffix}`,
+        extractId: `${options.extractId}${options.modulesSuffix}`,
+        css: {
+          modules: options.modules,
+          importLoaders: 1
```

Comment:
I know you did not change this line, but is it correct? See `importLoaders` for the css rule above.

Why do the rules have different importLoaders values @eliperelman?

Reply:
@eirikurn when using css-loader, `importLoaders` should equal the number of additional loaders (eg postcss, sass, etc) to be added before css-loader.

However, I'm _not_ sure why we're also incrementing importLoaders for `modules: true`. @eliperelman did that bit.

Reply:
It does appear they are doing this here: https://github.com/css-modules/webpack-demo/blob/master/webpack.config.js#L21, though I can't find any docs on whether importLoaders should be incremented when using css modules.

Reply:
I just did some tests and didn't seem to see a difference if the modules rule had importLoaders incremented or not…curious to see if @eliperelman knows where this came from.